group.yml

```diff
@@ -14,9 +14,7 @@ compliance:
 
 arches:
 - x86_64
-- ppc64le
 - s390x
-- aarch64
 
 multi_arch:
   enabled: true
```

Comment:
@joepvd I thinkwe problably want to pick up this change from Daniele's PR too. https://github.com/openshift-eng/ocp-build-data/pull/4975/files#diff-005fe59037264b0a0c3005a1c55f7043bdfcfab6c740a26bf8d9cb9ea3750f88R19